.git-blame-ignore-revs

```diff
@@ -0,0 +1,8 @@
+# Since version 2.23 (released in August 2019), git-blame has a feature
+# to ignore or bypass certain commits.
+#
+# This file contains a list of commits that are not likely what you
+# are looking for in a blame, such as mass reformatting or renaming.
+
+# Apply black formatting to the whole project
+f042a6bf6bd060ba8836a56b6939b77a662180cf
```

Comment:
[question] I've never used this feature, I'm wondering if we would want to wait until this PR is ready to merge, squash it into a single commit, and then add _that_ commit to this file. This would be to be able to ignore this entire effort at once.

Reply:
Good point, will move this out of the PR so we can add it after the eventual merge.

Reply:
Removed this - we can simply add the final commit hash after merging the PR indeed